### ISSUE TITLE
Bring back bold fonts on parts of the Concourse web UI

### ISSUE THREAD

**Issue by jama-pivotal** [concourse/concourse#3005]
This is related to the conclusion of #2938 

Bring back some of the boldness of the fonts in the build logs and the resource page. 

In my sketch mocks, I am using an Inconsolata bold so if we could update a couple spots with some bold style to Inconsolata I think we are there!

Can we add `font-weight: 700;` ( or however you decide to implement bold style) to the body font on the following?:

### Pipeline view
_screenshot without style_ 

![screen shot 2019-01-07 at 11 36 53 am](https://user-images.githubusercontent.com/27746614/50780835-f276ff00-1271-11e9-9559-e456cad1b8ae.png)

_with   ` font-weight: 700;`_
![screen shot 2019-01-07 at 11 37 29 am](https://user-images.githubusercontent.com/27746614/50780789-d83d2100-1271-11e9-881c-f7ad1dc90ba8.png)

### Breadcrumbs
_screenshot without style_ 

![screen shot 2019-01-07 at 11 40 03 am](https://user-images.githubusercontent.com/27746614/50780468-fe15f600-1270-11e9-9e76-fa2be233dcc2.png)

_with   ` font-weight: 700;`_

![screen shot 2019-01-07 at 11 39 55 am](https://user-images.githubusercontent.com/27746614/50780475-02421380-1271-11e9-9574-f433bfd9c0c8.png)

### Resource page

![screen shot 2019-01-07 at 11 30 09 am](https://user-images.githubusercontent.com/27746614/50779913-a0cd7500-126f-11e9-9c5b-15f5641f3ef7.png)
_L screenshot without style  R screenshot with `font-weight: 700;`_

### Build page logs
This is also realted to https://github.com/concourse/concourse/issues/2765

On the Build page, the log output texts need a bit more contrast with some bold added to the font style. Adding the bold to the style attribute of `timestamped-content` td (in addition to adding it to the body) will help  improve the readability of the logs. 

![screen shot 2019-01-07 at 10 55 28 am](https://user-images.githubusercontent.com/27746614/50779750-23a20000-126f-11e9-94b5-d6f92eae0cfb.png)

_L screenshot without style  R screenshot with `font-weight: 700;`_

**Comment by pivotal-jamie-klassen** [comment]
it is probably not a good idea to make the build logs have weight 700, because the logs themselves may make meaningful use of a distinction between normal and bold font. Google Fonts only provides two weights (400 and 700) for Inconsolata, so using 700 for the default weight will mean we have no "bolder" to go for cases when the logs themselves are bold.

@Lindsayauchin @vito

**Comment by vito** [comment]
@pivotal-jamie-klassen Good point. I'd also be a little worried that it looks too bold depending on the machine. The screenshot actually looks too bold to me anyway.

IMO bold makes sense for headers and such but for build output and other normal/freeform text (like comments and annotations when we get to them) I wouldn't bother.
